modules/installation-network-user-infra.adoc

```diff
@@ -96,17 +96,21 @@ network topology requirements.
 for platform containers and provide telemetry data to Red Hat.
 ====
 
-.Load balancers
+== Load balancers
 
-Before you install {product-title}, you must provision two layer-4 load
-balancers. The API requires one load balancer and the default Ingress Controller
-needs the second load balancer to provide ingress to applications.
+Before you install {product-title}, you must provision two load balancers. The requirements for these load balancers are as follows:
 
+1. *API Load Balancer*: Provides a common endpoint for users (both human and machine) to interact with and configure the platform.
+  * Layer 4 load balancing only (this is sometimes referred to as Raw TCP, SSL Passthrough, or SSL Bridge mode)
+  * Round robin load balancing
```

Comment:
@sttts , hi Stefan, could you double check above two lines whether looking good or needing update? I see https://github.com/openshift/machine-config-operator/pull/1670/files says "Google cloud load balancer is a L3LB that is special", not "Layer 4" here. Thanks!

Reply:
I totally miss what is described https://github.com/openshift/installer/blob/master/docs/dev/kube-apiserver-health-check.md.

Reply:
https://www.quora.com/What-is-the-difference-between-layer-3-and-layer-4-load-balancing-Why-is-layer-7-LB-used-inspite-of-its-drawbacks-of-being-a-bottleneck#:~:text=A%20layer%203%20load%2Dbalancer,and%20TCP%20or%20UDP%20ports.&text=Layer%207%20load%20balancer%20operates,actual%20content%20of%20each%20message.

Sounds like L4 is correct. But this is really a detail IMO. @squeed has more background.